src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/custom.py

```diff
@@ -57,70 +52,117 @@ def iot_hub_list(client, resource_group_name=None):
         return client.list_by_resource_group(resource_group_name)
 
 
-def iot_hub_show_connection_string(client, hub_name=None, resource_group_name=None, policy_name='iothubowner'):
```

Comment:
I think the original naming makes more sense and is consistent with terminology elsewhere in the CLI.